netbout-web/src/main/java/com/netbout/rest/TkStart.java

```diff
@@ -67,13 +82,35 @@ public TkStart(final Base bse) {
 
     @Override
     public Response act(final Request req) throws IOException {
-        final Inbox inbox = new RqAlias(this.base, req).alias().inbox();
+        final Href href = new RqHref.Base(req).href();
+        final Iterator<String> token = href.param("token").iterator();
+        String tokenkey = "";
```

Comment:
@prahladyeri Let's call this `key`, we should [avoid compound names](http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html).